framework/src/Volo.Abp.Data/Volo/Abp/Data/ConcurrencyStampExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using Volo.Abp.Domain.Entities;
+
+namespace Volo.Abp.Data
+{
+    public static class ConcurrencyStampExtensions
+    {
+        public static void SetConcurrencyStamp(this IHasConcurrencyStamp entity, string concurrencyStamp)
```

Comment:
```suggestion
        public static void SetConcurrencyStampIfNotNull([NotNull] this IHasConcurrencyStamp entity, [CanBeNull] string concurrencyStamp)
```

Reply:
Commit 96af9b1ff7fffc31c912825aa9288ee9d4961993 resolves it